src/components/stepper/StepperButton.js

```diff
@@ -34,6 +30,7 @@ StepperButton.propTypes = {
    * Handler function to receive updates when the value changes
    */
   onPress: PropTypes.func,
+  testID: PropTypes.string
```

Comment:
no need to declare the prop, it's a common prop

Reply:
I know. wired, I removed all these declarations...